src/components/RichTextEditor/Plugins/EmojiPlugin/EmojiPicker/EmojiPicker.tsx

```diff
@@ -54,13 +54,13 @@ export const EmojiPicker = ({
             </EmojiPickerSearchBar>
             <EmojiPickerContent
                 i18n={i18n}
-                setEmoji={setEmoji}
                 emojiLibrary={emojiLibrary}
                 isSearching={isSearching}
                 searchResult={searchResult}
                 visibleCategories={visibleCategories}
-                selectEmoji={selectEmoji}
                 settings={settings}
+                onSelectEmoji={onSelectEmoji}
```

Comment:
Won't these changes be breaking changes since we are changing the props names?

Reply:
No, it is internal and we are not using it anywhere yet. So it wont break anything